sdk/src/math/spotBalance.ts

```
@@ -382,35 +382,61 @@ export function calculateInterestRate(
 	delta = ZERO,
 	currentUtilization: BN = null
 ): BN {
+	// todo: ensure both a delta and current util aren't pass?
```

?

---

will silently ignore delta if `currentUtilization` is passed